### ISSUE TITLE
NVDA doesn't read properly PPT files witch contain text with transition effects

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2907]
**Reported by k_kolev1985 on 2013-01-01 20:23**
Hello,

I've recently came across a PPT file, witch has effects for the text in it. What I mean is that the appearing of the text on the slides is with an effect. And sometimes the text for a slide doesn't appear all at once, but segment after a segment. And for each segment to appear, the user has to press "space" or "Page Down" keys. And NVDA doesn't read correctly such a document in full-screen mode - most of the time, it gives audible error sounds.

I'm using MS Office 2007SP3 in bulgarian and Windows 7 Ultimate SP1 32-bit in bulgarian.

I'm attaching the problematic PPT file and my NVDA log file.

**Comment by Qchristensen** [comment]
Here are the original files from this ticket (log file renamed to .txt and ppt file renamed to pptx but not converted to new format - it loads fine in PowerPoint 2016).  I've also attached a test file I made with four slides with several different text animation effects rather than slide transition effects (the issue is named "transition effects" however that term is used for how the view transitions from one slide to the next whereas the description seems to more closely match "animations" - how text is set to appear on each slide (as text can be set to not all appear as soon as a slide loads).

Finally in terms of this issue, I don't get any error beeps but simply that NVDA reads the full contents of any slide as soon as it loads, even if the contents are not yet shown visually.  Nothing is read as I press spacebar, enter or page down to progressively show the bullet points on the slide. ( cc @michaelDCurran )


[nvda-k_kolev1985-1 log.txt](https://github.com/nvaccess/nvda/files/280781/nvda-k_kolev1985-1.log.txt)
[various transition effects.pptx](https://github.com/nvaccess/nvda/files/280782/various.transition.effects.pptx)

[Kompjutyrni virusi i predpazvane ot tjah.pptx](https://github.com/nvaccess/nvda/files/280784/Kompjutyrni.virusi.i.predpazvane.ot.tjah.pptx)

**Comment by Qchristensen** [comment]
quick note, you'll need to rename the original file Kompjutyrni virusi i predpazvane ot tjah.pptx back to .ppt as PowerPoint can't figure out a ppt file with pptx extension.

**Comment by Adriani90** [comment]
@k_kolev1985 is this still reproducible with NVDA 2018.3 or with last beta or Alpha?

**Comment by Qchristensen** [comment]
NVDA no longer gives errors as originally reported in this issue.  The wider issue of NVDA reading all text of slides with text animations as soon as the slide is loaded, even though, for instance, each bullet point on the slide only appears as the spacebar is pressed, still exists.  I will close this issue now, in favour of keeping further work and discussion on that topic in Issue #7877 which describes the technicalities of the issue in more depth.
